packages/rum-core/src/domain/resource/retrieveInitialDocumentResourceTiming.ts

```diff
@@ -0,0 +1,46 @@
+import type { RelativeTime } from '@datadog/browser-core'
+import { assign, runOnReadyState } from '@datadog/browser-core'
+import { supportPerformanceTimingEvent, RumPerformanceEntryType } from '../../browser/performanceObservable'
+import type { RumPerformanceResourceTiming } from '../../browser/performanceObservable'
+import type { RumConfiguration } from '../configuration'
+import { getDocumentTraceId } from '../tracing/getDocumentTraceId'
+import { FAKE_INITIAL_DOCUMENT, computeRelativePerformanceTiming } from './resourceUtils'
+
+export function retrieveInitialDocumentResourceTiming(
```

Comment:
**💭 thought:** ‏I think we can "clean" this up and don't fake a performance entry like this. Are you planning to do it when we'll migrate the `NAVIGATION` performance entry to the new `createPerformanceObserver`?

Reply:
I thought of cleaning it but the PR is already quite substantial. Like you mention, let's discuss it in the following NAVIGATION PR ;)